### PR TITLE
Add spec_url for H to L CSS properties

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: css/properties/height.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
       "height": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/height",
+          "spec_url": "https://drafts.csswg.org/css-sizing-4/#width-height-keywords",
```
Comment (Elchi3) [inline css/properties/height.json:7]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-sizing/#preferred-size-properties",
```

File: css/properties/ime-mode.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
       "ime-mode": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/ime-mode",
+          "spec_url": "https://drafts.csswg.org/css-ui-3/#input-method-editor",
```
Comment (Elchi3) [inline css/properties/ime-mode.json:7]:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-ui/#input-method-editor",
```

File: css/properties/list-style-type.json [hunk 3]
```diff
@@ -4,6 +4,10 @@
       "list-style-type": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/list-style-type",
+          "spec_url": [
+            "https://drafts.csswg.org/css-counter-styles/#extending-css2",
+            "https://drafts.csswg.org/css-lists/#propdef-list-style-type"
```
Comment (Elchi3) [inline css/properties/list-style-type.json:9]:
```suggestion
            "https://drafts.csswg.org/css-lists/#propdef-list-style-type",
            "https://drafts.csswg.org/css-counter-styles/#extending-css2"
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Some nits, LGTM otherwise.

I swapped one order as I think the main spec should be first. (if MDN renders spec sections from this data, it would take items in the order of this array, I guess).

**Comment by sideshowbarker** [comment]
> I swapped one order as I think the main spec should be first. (if MDN renders spec sections from this data, it would take items in the order of this array, I guess).

yeah, makes sense

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**